examples/text-generation/quantization_config/act_maxabs_pow2_weights_pcs_opt_pow2_quant.json

```diff
@@ -0,0 +1,10 @@
+{
+    "method": "HOOKS",
+    "mode": "QUANTIZE",
+    "observer": "maxabs",
+    "scale_method": "ACT_MAXABS_POW2_WEIGHTS_PCS_OPT_POW2",
+    "allowlist": {"types": [], "names":  []},
+    "blocklist": {"types": [], "names":  []},
+    "dump_stats_path": "./hqt_output/measure",
+    "dump_stats_xlsx_path": "./hqt_output/measure/fp8stats.xlsx"
+}
```

Comment:
why do we need this change? do we need to update any readme for this?

Reply:
in version 1.15  we changed this file scaling method to:
"scale_method": "ACT_MAXABS_POW2_WEIGHTS_PCS_OPT_POW2",
We switched to this scaling method, which is not hardware-aligned to guadi as the previous scaling method was, since it returned more accurate results when quantizing the model weights per channel (per-channel-quantization: PCQ).
however the file name remained with the old scale method, this commit aligns the file name that will match the scale method (just a syntax change to avoid confusion that file name does not match the content in it).
Im not sure regarding needed documentation changes. 
You can add Eran Geva to the reviewers, maybe he will have more insights on that.

Reply:
PCQ is not part of the readme, however it is documented in the GaudiDocs